frontend/src/components/form/BookingsSearch.tsx

```diff
@@ -1,106 +1,134 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react';
 
-import moment from 'moment'
+import moment from 'moment';
 
-import {
-  Box,
-  Grid,
-} from '@material-ui/core'
+import { Box, Grid } from '@material-ui/core';
 
 import {
   StyledFormControl,
   StyledLabel,
   StyledTextField,
   StyledFormLabel,
   GridRow,
-} from './inputs/DirectoryInputs'
+} from './inputs/DirectoryInputs';
 
-import { Schema, Initial } from 'components/form/schemas/booking-search.schema'
-import { ButtonPrimary } from 'components/Buttons'
-import BookingsRange from 'components/form/BookingsRange'
+import { Schema, Initial } from 'components/form/schemas/booking-search.schema';
+import { ButtonPrimary } from 'components/Buttons';
+import BookingsRange from 'components/form/BookingsRange';
 
-import { ErrorMessage, Field, Form, Formik, FormikProps } from 'formik'
+import { ErrorMessage, Field, Form, Formik, FormikProps } from 'formik';
+import { getLocationDetails } from 'util/locationFetch';
+import AutocompleteInput from './inputs/Autocomplete';
 
-const dateFormat = 'MMM D, YYYY'
+const dateFormat = 'MMM D, YYYY';
 
-export default function Search({ getSearchResults }: { getSearchResults: Function }) {
+export default function Search({
+  getSearchResults,
+}: {
+  getSearchResults: Function;
+}) {
+  const [locations, setLocations] = useState(['Loading...']);
+  useEffect(() => {
+    async function fetchLocation() {
+      const fetchedLocations: string[] = await getLocationDetails();
+      setLocations(fetchedLocations);
+    }
+    fetchLocation();
+  }, [setLocations]);
```

Comment:
I think maybe refer to `InterpreterSearch.tsx`, I already change not use "string", then you can send `id` to backend.
to avoid search by name.

![image](https://user-images.githubusercontent.com/46575401/115918485-6d0cbe00-a42c-11eb-8920-434c24e3986e.png)

Reply:
and `AutoComplete` part. can `setFieldValue: location_id`

![image](https://user-images.githubusercontent.com/46575401/115918864-e73d4280-a42c-11eb-9ce0-7d0ad5a2e701.png)

Reply:
I will check